pkg/ref/api_id.go

```diff
@@ -0,0 +1,23 @@
+package ref
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Parse parses the steve api ID.
+func Parse(ref string) (namespace string, name string) {
+	parts := strings.SplitN(ref, ":", 2)
+	if len(parts) == 1 {
+		return "", parts[0]
+	}
+	return parts[0], parts[1]
+}
```

Comment:
Looks like steve API ID is in the format of `default/name`, so we should either change the function description or change the format we are using

Reply:
`Parse` method will change other UI APIs, I prefer to update the `Construct` method in this PR, and use another new PR to repair the `Parse` method, cc @aiwantaozi .